src/main/web/src/organisms/header/Notifications.tsx

```diff
@@ -1,45 +1,84 @@
-import React, { useEffect, useState } from 'react';
+import React, {Dispatch, SetStateAction, useEffect, useMemo, useState} from 'react';
+import { NavigateFunction, useNavigate } from "react-router-dom";
+import "./Notifications.css";
+import config from "../../config/config";
+import {getJWT} from "../../services/AuthService";
 
 interface NotificationsProps {
   showNotifications: boolean;
+  notifications: NotificationModel[];
+  setNotifications: Dispatch<SetStateAction<NotificationModel[]>>;
 }
 
-interface NavigatorWithBadge extends Navigator {
-  setAppBadge?: (count: number) => Promise<void>;
+export interface NotificationModel {
+  notificationId: number;
+  groupId: number;
+  text: string;
+  link: string;
+  type: string;
 }
 
-export const Notifications: React.FC<NotificationsProps> = ({ showNotifications }) => {
-  const [notificationShown, setNotificationShown] = useState(false);
+interface DeleteNotificationModel {
```

Comment:
Best practise: Extract this to a seperate file called `DeleteNotificationModel` in a folder `models`